### DISCUSSION TITLE
How can I put the adsense banner above the jupyter notebook pages?

### DISCUSSION THREAD

**Discussion by hellojinwoo** [squidfunk/mkdocs-material#5430] [Q&A]
- I am using `mkdocs-jupyter` plugins to put my jupyter notebook files on the mkdocs blog.
- I am curious to know if it is possible to put the adsense banners at the top of the jupyter notebook pages (i.e. above the jupyter notebook pages) when it is rendered.
- If it is possible, would there be a tutorial or a guide to let me know how this can be done? Any excerpt of codes would be greatly appreciated too

**Comment by hellojinwoo** [comment]
Found this thread https://github.com/squidfunk/mkdocs-material/issues/1311 so closed this discussion

**Comment by hellojinwoo** [comment]
Sorry @squidfunk , but can you kindly tell me how to add the banner at the top of the page, which is created with mkdocs-jupyter? I have tried for several hours, but ended up failing at it. :(

**Reply by hellojinwoo** [reply]
This is what I have tried to put an adsense banner at the top of the webpage created with `mkdocs-jupyter` but didn't work.

```
1. Create a folder named templates in your MkDocs project, if it doesn't already exist.
2. Inside the templates folder, create a new file named custom_notebook.html.
3. In the custom_notebook.html file, extend the base.html template and override the content block with your AdSense code:

{% extends "base.html" %}

{% block content %}
  <!-- Google AdSense Code -->
 ... (I put mt google adsense code here. Didn't include it here for a security reason)
  <!-- End Google AdSense Code -->

  <!-- Render the notebook content -->
  {{ super() }}
{% endblock %}

4. Update the mkdocs.yml configuration file to use the custom template:
plugins:
  - search
  - mkdocs-jupyter:
      exclude: .ipynb_checkpoints
      template: templates/custom_notebook.html
```

**Reply by squidfunk** [reply]
I'm sorry, but I can't help each and every user with their customization. I recommend asking the maintainers of mkdocs-jupyter, as it seems related to this plugin (which we don't maintain) or on StackOverflow. Other than that, maybe somebody else has some time to look into this very specific request.
